armi/operators/operator.py

```diff
@@ -567,17 +567,20 @@ def _expandCycleAndTimeNodeArgs(self, interactionName):
         """
         if interactionName == "Coupled":
             cycleNodeInfo = (
-                f" - timestep: cycle {self.r.p.cycle}, node {self.r.p.timeNode}"
-                f" - iteration {self.r.core.p.coupledIteration}"
+                f" - timestep: cycle {self.r.p.cycle}, node {self.r.p.timeNode}, "
+                f"year {self.r.p.time} - iteration {self.r.core.p.coupledIteration}"
```

Comment:
These time values are going to be really ugly decimals, I think. Should we force them to 2 decimals, or something nicer?

Reply:
Ah! I didn't see that in our unit tests because of the simplicity of unit tests. That's a really good call, let me go do that.

Reply:
Done!